### DISCUSSION TITLE
How to remove every data from Zen browser

### DISCUSSION THREAD

**Discussion by speedy1804** [zen-browser/desktop#2146] [Q&A] [answered]
Hey, I would like to completely remove Zen Browser and its data from my PC and perform a clean reinstallation. I tried to uninstall the browser with the Revo uninstaller (to delete all data) - the uninstallation worked, but all data was still there when I downloaded the browser again. Can anyone tell me how I can really delete all data, i.e. practically reinstall the browser? I am open to solutions, thank you very much!

**Comment by speedy1804** [comment, marked as answer]
got it

**Reply by crxssed7** [reply]
How did you do it?
